src/components/layouts/ecosystem/Iqwiki.tsx

```diff
@@ -12,9 +11,12 @@ const Iqwiki = () => {
   const [appTheme, setAppTheme] = useState("");
 
   useEffect(() => {
-    setAppTheme(theme || "");
+    setAppTheme(theme ?? "");
   }, [theme]);
-  let imageUrl, imageUrl2, imageUrl3;
+  let imageUrl: string | undefined = undefined;
+
```

Comment:
any reason for this?

Reply:
TS was complaining about having any type :))

Reply:
hmmm, I have raised it because reducing the lines of code can also improve performance. maybe that was before you assigned value to the variable. I reverted and it worked but ts complained about declaring more than one variable on a single line.

Reply:
Hmm 🤔, it’s the same process tho, the variable isn undefined then gets defined after.